src/components/index.tests.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import { Accordion } from './index'
+import { Accordion, Stepper } from './index'
```

Comment:
Is there a reason you've opted to import `Stepper` and `Accordion` in the same line?

Reply:
Lol, no 🤦 I'll fix that thanks Steph

Reply:
I'm actually suspicious that vscode auto did that lol

Reply:
fixed in v1.1.187